src/core/Microsoft.PowerToys.Settings.UI/Views/FancyZonesPage.xaml

```diff
@@ -70,18 +70,30 @@
                     <TextBlock Margin="12,0,0,0" x:Uid="FancyZones_LaunchEditorButtonControl"/>
                 </StackPanel>
             </Button>
-            
+
             <TextBlock x:Uid="FancyZones_ZoneBehavior_GroupSettings"
                        Style="{StaticResource SettingsGroupTitleStyle}"
                        />
 
+            <StackPanel Orientation="Horizontal" Margin="{StaticResource SmallTopMargin}">
+                <TextBlock x:Uid="FancyZones_HotkeyEditorControl"
+                           ToolTipService.ToolTip="Only the shortcuts with the following hotkeys are valid: Win, Ctrl, Alt, Shift."
+                           Text="Editor Hotkeys"/>
+                <Viewbox Height="{StaticResource MediumFontSize}" Width="{StaticResource MediumFontSize}" Margin="5,2,0,0" >
+                    <PathIcon
+                        VerticalAlignment="Center"
+                        ToolTipService.ToolTip="Only the shortcuts with the following hotkeys are valid: Win, Ctrl, Alt, Shift."
+                        Data="M960 1920q-133 0-255-34t-230-96-194-150-150-195-97-229T0 960q0-133 34-255t96-230 150-194 195-150 229-97T960 0q133 0 255 34t230 96 194 150 150 195 97 229 34 256q0 133-34 255t-96 230-150 194-195 150-229 97-256 34zm0-1792q-115 0-221 30t-198 84-169 130-130 168-84 199-30 221q0 114 30 220t84 199 130 169 168 130 199 84 221 30q114 0 220-30t199-84 169-130 130-168 84-199 30-221q0-114-30-220t-84-199-130-169-168-130-199-84-221-30zm-64 640h128v640H896V768zm0-256h128v128H896V512z"/>
+                </Viewbox>
+            </StackPanel>
+
             <CustomControls:HotkeySettingsControl 
-                                          x:Uid="FancyZones_HokeyEditorControl_Header"
                                           Width="240"
                                           HorizontalAlignment="Left"
-                                          Margin="{StaticResource SmallTopMargin}"
+                                          ToolTipService.ToolTip="Only shortcuts with the following hotkeys are valid: Win, Ctrl, Alt, Shift."
+                                          Margin="0,8,0,0"
```

Comment:
Still seems to be a bigger gap. Left (default), right is custom header.

I'm guessing it's either 5 or 6 pixels. Weird that did didn't render that way in my custom XAML snippet. Sorry for that!
![image](https://user-images.githubusercontent.com/9866362/83809633-1af96780-a6b7-11ea-9a73-ca2514f394eb.png)

Reply:
I guess we need to change the margin from 8?

Reply:
changed the size to 5